src/Buffer.h

```diff
@@ -49,19 +49,30 @@ D3D11_BUFFER_DESC ConstantBufferDesc(bool dynamic = false)
 class ConstantBuffer
 {
 public:
-	explicit ConstantBuffer(D3D11_BUFFER_DESC const& a_desc) :
-		desc(a_desc)
+	explicit ConstantBuffer(D3D11_BUFFER_DESC const& desc) :
```

Comment:
BTW: I notice most of our constant buffer are created without the Dynamic flag set (e.g. GraphCollision), which means it should be much costly when we update them, wondering if that is intentional?

Reply:
That was an oversight on my part. I will fix that thanks. I could not measure a performance difference though.